src/tests/ftest/util/dmg_utils.py

```diff
@@ -1340,9 +1340,9 @@ def get_json_response(data, key, description):
     """
     response = {}
     try:
-        if data['error']:
+        if 'error' in data and data['error']:
             raise CommandFailure("{} failed: {}".format(description, data['error']))
-        if len(data['response']['host_errors']) > 0:
+        if 'response' in data and len(data['response']['host_errors']) > 0:
```

Comment:
After discussing this with Dalton, we recommend:
```suggestion
        if 'host_errors' in data['response'] and len(data['response']['host_errors']) > 0:
```

Reply:
Done.